README.md

```diff
@@ -70,45 +70,44 @@ for more information.
 
 Our goal is to have extensive use-case and functional documentation.
 
-See the [main Kubernetes docs](https://kubernetes.io/docs/concepts/service-catalog/)
-and additional documentation [here](./docs/v1).
+See the [Service Catalog documentation](https://kubernetes.io/docs/concepts/service-catalog/)
+on the main Kubernetes site, and the [project documentation](./docs/README.md) here on GitHub.
 
-See [here](https://github.com/kubernetes-incubator/service-catalog/wiki/Examples) for examples and
-[here](https://github.com/openservicebrokerapi/servicebroker/blob/master/gettingStarted.md) for
-broker servers that are compatible with this software.
+The project Wiki will contain [examples](https://github.com/kubernetes-incubator/service-catalog/wiki/Examples),
```

Comment:
Since the entirety of the contents on that page is a redirect to the osbapi repo, link directly there instead.

Reply:
Actually, it struck me as odd to link to an examples page that contained no examples, so I've removed that bit altogether. Once we get some samples on that Wiki page, I'll add the mention and link back in.

Cc: @arschles